pelicun/tools/DL_calculation.py

```diff
@@ -633,6 +633,10 @@ def _parse_config_file(  # noqa: C901
 
         log_msg('Trying to auto-populate')
 
+        # Add the demandFile to the config dict to allow demand dependent auto-population
+        update(config, '/DL/Demands/DemandFilePath', demand_file)
+        update(config, '/DL/Demands/SampleSize', str(realizations))
+
```

Comment:
These values are added to config in lines 637-638, but they are not copied to config_ap. To remove lines 670-671, I copied them from config to config_ap in auto.py.

Reply:
Yes, I can see how that happens. It could be fixed by asking for a more sophisticated auto-population script or by merging the auto-populated AIM file with the original in a more complex and careful way. Both options seem worse than keeping two extra lines of code for now. So, let's leave this as is and come back to it if the number of parameters we need to embed in the DL before auto population grows.